sessionctx/binloginfo/binloginfo.go

```diff
@@ -108,3 +111,19 @@ func SetDDLBinlog(client interface{}, txn kv.Transaction, jobID int64, ddlQuery
 	}
 	txn.SetOption(kv.BinlogInfo, info)
 }
+
+const specialPrefix = `/*!90000 `
+
+func addSpecialComment(ddlQuery string) string {
+	if strings.Contains(ddlQuery, specialPrefix) {
+		return ddlQuery
+	}
+	upperQuery := strings.ToUpper(ddlQuery)
+	reg, err := regexp.Compile(`SHARD_ROW_ID_BITS\s*=\s*\d+`)
+	terror.Log(err)
```

Comment:
If err isn't nil, reg may be nil.

Reply:
It's just added to pass `errcheck`, it never returns nil